docs/root/operations/admin.rst

```diff
@@ -190,9 +190,118 @@ The fields are:
 
   .. http:get:: /stats?format=json
 
-  Outputs /stats in JSON format. This can be used for programmatic access of stats.
+  Outputs /stats in JSON format. This can be used for programmatic access of stats. Counters and Gauges
+  will be in the form of a set of (name,value) pairs. Histograms will be under the element "histograms",
+  that contains "supported_quantiles" which lists the quantiles supported and an array of computed_quantiles
+  that has the computed quantile for each histogram. Only histograms with recorded values will be exported.
 
-  .. http:get:: /stats?format=prometheus
+  If a histogram is not updated during an interval, the ouput will have null for all the quantiles.
+  
+  Example histogram output:
+
+.. code-block:: json
+
+  {
+    "histograms": {
+    "supported_quantiles": [
```

Comment:
the indentation in this description looks wrong.  This line should be indented one more level than the previous one.  Do you think it also makes sense, doc purpose, to put these 9 quantiles each on one line?